test/test_sparse.py

```diff
@@ -2678,6 +2679,103 @@ def sparse_log(x):
         test_op(3, 100, [3, 4, 2, 3, 5, 2])
         test_op(4, 100, [3, 4, 2, 3, 5, 2])
 
+    @cpu_only
+    def test_sparse_matmul(self):
+
+        def _indices2csr(indices, dim):
+            nnz = len(indices)
+            r = [0] * (dim + 1)
+            last_i = 0
+            for i in indices:
+                if i == last_i:
+                    r[last_i + 1] += 1
+                else:
+                    for _i in range(last_i, i + 1):
+                        r[_i + 1] = r[last_i + 1]
+                    last_i = i
+                    r[last_i + 1] += 1
+            for _i in range(last_i, dim):
+                r[_i + 1] = r[last_i + 1]
+            assert r[-1] == nnz
+            return r
+
+        def sparse_mm(a, b):
```

Comment:
Is there a SciPy reference this test can use instead of a custom Python reference?